templates/modulesTakenPriorToOthers.html

```diff
@@ -0,0 +1,100 @@
+$def with (modulePairs, allAySems, studentCounts, studentPriorCount, moduleA, moduleB, targetAySem, studentsInModuleB)
+
+$var title:View Modules Taken Prior to Others
+
+$if modulePairs is not None:
+	<div class="container-fluid">
+		<h1 class="text-center"><b>Modules that are taken prior to other modules</b></h1>
+		<p class="text-center">Shows all pairs of modules where module A is taken prior to module B</b></p>
+		<br>
+		<div class="modules-taken-prior-search-form">
+			<form action="/moduleTakenPriorToOthers" method="POST">
+				<p>Tell me the number of students who have taken</p>
```

Comment:
"Tell me the number of students who have taken **A** before they are *going to* take **B** in **a specified AY-Semester**"

I tried injecting a <code>studentplan</code> row with the following: ['D5000000A', true, 'CS1020', 'AY 16/17 Sem 2']
(meaning student D5000000A has already taken CS1020 in AY 16/17 Sem 2)

This particular student was also taken into account for the query "Find students who have taken CS1010 before they are going to take CS1020 in AY 16/17 Sem 2".

Due to the **going to** clause, that student should not have been considered seeing that he has taken CS1020 in AY 16/17 Sem 2 **already**.

Thus, I wish to double-check whether students who HAVE TAKEN the module already should or should not be considered in this query.